src/ocean/text/formatter/SmartUnion.d

```diff
@@ -0,0 +1,205 @@
+/*******************************************************************************
+
+    Helper function for formatting a SmartUnion's active member to a string.
+
+    This function is not placed in the SmartUnion class itself so as to avoid
+    spreading the dependency on ocean.text.convert.Formatter throughout the
+    codebase.
+
+    Copyright:
+        Copyright (c) 2018 Sociomantic Labs GmbH.
+        All rights reserved.
+
+    License:
+        Boost Software License Version 1.0. See LICENSE_BOOST.txt for details.
+        Alternatively, this file may be distributed under the terms of the Tango
+        3-Clause BSD License (see LICENSE_BSD.txt for details).
+
+*******************************************************************************/
+
+module ocean.text.formatter.SmartUnion;
+
+import ocean.transition;
+import ocean.core.SmartUnion;
+import ocean.core.Verify;
+import ocean.text.convert.Formatter;
+
+/*******************************************************************************
+
+    Gets a wrapper struct with a toString method to format the provided smart
+    union.
+
+    Params:
+        SU = type of smart union to get formatting wrapper for
+        smart_union = smart union to get formatting wrapper for
+        include_name = flag to toggle formatting of the name of the active union
+            member. If true, format output will look like
+            "<activename>: value". If false, format output will look like
+            "value"
+
+*******************************************************************************/
+
+public SmartUnionFormatter!(SU) asActiveField ( SU ) ( SU smart_union,
+    bool include_name = false )
+{
+    return SmartUnionFormatter!(SU)(smart_union, include_name);
+}
+
+///
+unittest
+{
+    union U
+    {
+        int i;
+        cstring s;
+    }
+
+    SmartUnion!(U) su;
+
+    mstring buf;
+    // sformat from ocean.text.convert.Formatter
+    sformat(buf, "union active member: {}", asActiveField(su));
```

Comment:
The output is not tested ?
Also might as well use `format` and remove the buffer.

Reply:
It's a usage example, not a test.